go_cli.py

```diff
@@ -231,14 +231,23 @@ def get_history_commands(history_file):
     # Generate a unique interaction ID
     interaction_id = str(uuid.uuid4())
 
+
+    personalized_input = f"""
+    Some relevant context about my history:
+    {personalize(user_input, get_history_commands(HISTORY_FILE), False)}
+
+    The query of the user is:
+    {user_input}
+    """
+
     if args.history:
         commands = get_history_commands(HISTORY_FILE)
     else:
         with Halo(text=f"{GORILLA_EMOJI}Loading", spinner="dots"):
             try:
                 data_json = {
                     "user_id": user_id,
-                    "user_input": user_input,
+                    "user_input": personalized_input,
```

Comment:
Maybe we have another field for "personalized_input"? This means I might also have to update the server side.